README.adoc

```diff
@@ -493,6 +493,12 @@ api:
     - etc..
 ----
 
+|`AUTH_STRATEGY`
+|The authentication strategy to be used by Kiali. The possible values are `none`, for no checking of credentials, `oauth` for using Openshift oauth, and `login` for the default login page (default).
```

Comment:
We might want to rename this to something like 'openshift' or 'openshift-oauth'. We might want to support other oauth types in the future.